### PR TITLE
Create latn_qwertz_fr_CH.xml

### PR DESCRIPTION
**Pull request by ElucGeek**
Adapted from qwertz DE to match qwertz fr_CH layout.

### CODE REVIEW

**Comment by Julow** [comment]
Thanks! I updated the generated files and added the locale for fr-CH.

The empty spaces close to the shift and backspace keys are huge. I suggest making them smaller.
shot

(don't mind the dead keys for the accents, I plan to make them disappear before the next release)

I can leave this PR open for a while, so you can easily make changes to the layout.

**Comment by ElucGeek** [comment]
Hi, thanks for the correction and the preview. I'm not really a dev so I was not sure I did it correctly.
I really kept the QWERTZ DE layout and change the accents for the french ones, first I tried to stay as close to the SN standard but it's really not the best layout you could imagine, see here for more information: https://www.farah.cl/Keyboardery/A-Visual-Comparison-of-Different-National-Layouts/#CH . 

So I took the liberty to have ç at the bottom of c, as it's used a lot in French but on fr_ch layout it's done with shift+4, which make no sense at all. To be closer to SN it could be at the top left of r.
Same for the ê, which is done by ^+e, basically using dead key. In fact a,e,i,o,u could all have ^ on top, so we need to keep the dead key somewhere, I have to change that.

What do you suggest for the spaces, to make shift and enter with a width of 2 instead of 1.5? Alternatively I could create a key at the right of m. What is the closest in the fr_ch layout is . and , keys, instead on having them on bottom left of b and n. I could as well do a key with ê (most used ^) and have all other around it â top left, î top right,... But I'm not sure it's good to have this right next to the backspace (delete) key.

Is there I way I can test the custom layout directly (without compiling the app myself)? I saw there is a custom layout option but where should I enter the code then?

By the way, why is the enter key in the screenshot "Fin"? Couldn't this be the return symbol?

**Comment by Julow** [comment]
It's better to diverge a little from standards to make the layout more usable. Standards tends to cram many symbols on the number row and on the right of the keyboard, two places that layouts don't have in this app.

It's reasonable to show the dead key for ê but the accented letters directly for éèà, etc.. as it's used slightly less often.

A width of 2 is what I'd prefer for shift and enter, but I'm not a user of this layout. A width of 1.75 and a shift of 0.25 would already look better.

**Comment by Julow** [comment]
I fixed the conflicts by pushing to your branch. A syntax error remains.
I would like to merge this soon.

File: res/xml/latn_qwertz_fr_ch.xml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <key key0="é" key1="ë" key2="ö"/>
+    <key key0="à" key1="ï" key2="ä"/>
+  </row>
+  <row>
+    <key width="1.5" key0="shift" key2="loc capslock"/>
+    <key key0="y" key1="&gt;" key2="|" key3="&lt;"/>
+    <key key0="x"/>
+    <key key0="c" key3="ç"/>
+    <key key0="v"/>
+    <key key0="b" key1=";" key3="," key4"₿"/>
```
[the remark this answers is not in the document]
Comment (ElucGeek) [reply res/xml/latn_qwertz_fr_ch.xml:36]: Ok to remove it, I like it and use it but it might be too advance on our time to add it in keyboard layout.

**Comment by ElucGeek** [comment]
It's all fine for me, I have been testing the layout for some weeks now, it's quite good for my usage at least.
What should I do next? (sorry I'm not a dev, not sure what to do to push it to the main branch).

**Comment by Julow** [comment]
If the layout works for you then it can be merged!

Other currency symbols are on the `Fn` layer, so they are accessible from every latin layouts. You could add it here if you wish: https://github.com/Julow/Unexpected-Keyboard/blob/master/srcs/juloo.keyboard2/KeyModifier.java#L255

**Pull request merged**